docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+  web:
+    restart: always
+    build: ./server/web
+    ports:
+      - "8000:8000"
+    links:
+      - mysql:mysql
+    volumes:
+      - ./server/web/code:/code
+  
+  nginx:
+    restart: always
+    image: nginx:latest
+    ports:
+      - "80:80"
+    volumes:
+      - ./server/nginx:/usr/share/nginx/html
+      - ./server/nginx.conf:/etc/nginx/nginx.conf
+    volumes_from:
+      - web
+    links:
+      - web:web
+  
+  mysql:
+    restart: always
+    image: mysql:latest
+    ports:
+      - "3306:3306"
+    volumes:
+      - data_sql:/var/lib/mysql
+    environment:
+      MYSQL_DATABASE: "test"
+      MYSQL_USER: "test"
+      MYSQL_PASSWORD: "test"
```

Comment:
It probably doesn't matter for now since it's just 'test' all the way through, it's not like we're putting out anything sensitive. But yes we should move to envars soon. I'll open an issue but I think this is fine to push.